crates/ibc/src/core/ics04_channel/packet.rs

```diff
@@ -52,6 +52,18 @@ impl core::fmt::Display for PacketMsgType {
     }
 }
 
+#[cfg_attr(
+    feature = "parity-scale-codec",
+    derive(
+        parity_scale_codec::Encode,
+        parity_scale_codec::Decode,
+        scale_info::TypeInfo
+    )
+)]
+#[cfg_attr(
+    feature = "borsh",
+    derive(borsh::BorshSerialize, borsh::BorshDeserialize)
+)]
```

Comment:
Could we also add this to all **types that derive `serde::Serialize/Deserialize`** in modules:
+ `core::ics02_client::trust_threshold`
+ `core::ics03_connection::connection`
+ `core::ics03_connection::version`
+ `core::ics04_channel::channel`
+ `core::ics04_channel::commitment`
+ `core::ics04_channel::version`
+ `core::ics26_routing` (`ModuleId`)
+ `crate::Signer` type

Reply:
done